File: inflearn/java/2-02/Main.java
import java.util.*;

public class Main {
    public static int solution(int n, String str) {
        String[] arr = str.split(" ");
        int answer = 1;
        int a = Integer.parseInt(arr[0]);
        for (int i = 1; i < n; i++) {
            int b = Integer.parseInt(arr[i]);
            if (a < b) {
                a = b;
                answer++;
            }
        }
        return answer;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine(); // 입력 버퍼 클리어
        String input = sc.nextLine();

        System.out.println(solution(n, input));
    }
}

File: inflearn/javascript/2-02/main.js
const ps = require('../modules/ps');

const solution = (n, arr) => {
  let answer = 1; // 첫번째는 항상 보인다.
  let big = arr[0];
  for (let i = 1; i < n; i++) {
    if (big < arr[i]) {
      big = arr[i];
      answer++;
    }
  }
  return answer;
};

ps.main(async () => {
  let n = parseInt(await ps.readLine()); // 문자열을 입력 받아 정수 반환
  let arr = await ps.readArrayLine(); // 문자열을 입력 받아 Number 배열 형으로 반환

  console.log(solution(n, arr));
});
